Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/BuiltInLinuxWebAppDeploymentProvider.ts

```diff
@@ -94,8 +99,17 @@ export class BuiltInLinuxWebAppDeploymentProvider extends AzureRmWebAppDeploymen
 
     public async UpdateDeploymentStatus(isDeploymentSuccess: boolean) {
         if(this.kuduServiceUtility) {
```

Comment:
To minimize complexity level, instead of covering all function within a `if(this.kuduServiceUtility)` statement, you can add finalize method with `if(!this.kuduServiceUtility) { return }` statement. It will do same as far as I see though the code.